platform.js

```diff
@@ -695,7 +695,7 @@
     // Detect non-Opera (Presto-based) versions (order is important).
     if (!version) {
       version = getVersion([
-        '(?:Cloud9|CriOS|CrMo|Edge|FxiOS|IEMobile|Iron|Opera ?Mini|OPiOS|OPR|Raven|SamsungBrowser|Silk(?!/[\\d.]+$))',
+        '(?:Cloud9|CriOS|CrMo|Edge|EdgA|EdgiOS|FxiOS|IEMobile|Iron|Opera ?Mini|OPiOS|OPR|Raven|SamsungBrowser|Silk(?!/[\\d.]+$))',
         'Version',
```

Comment:
Is `EdgA` the full identifier?

Reply:
From Microsoft Info the UA for Edge For Android is like: ... EdgA/41.1.35.1
And for iOS is: ... EdgiOS/41.1.35.1

Reply:
Thank you!